README.md

```diff
@@ -9,6 +13,15 @@ The environment on the pi is really weird...
 
 To run simple scripted movement:
 
-`sudo /home/hera-led/Hera_games_env/bin/python scripts/test_movement.py`
+`sudo /home/hera-led/Hera_games_env/bin/python scripts/test_movement`
+
+`sudo /home/hera-led/Hera_games_env/bin/python muck_around/test_controler_input`
```

Comment:
I think this file is moved to `scripts` folder?

Reply:
Yeah. True. Fixed in new commit